lib/Data/Filesystem.php

```diff
@@ -228,7 +228,13 @@ public function readComments($pasteid)
                     $comment['parentid'] = $items[2];
 
                     // Store in array
-                    $key            = $this->getOpenSlot($comments, (int) $comment['meta']['created']);
+                    $key            = $this->getOpenSlot(
+                        $comments, (
+                            (int) array_key_exists('created', $comment['meta']) ?
+                            $comment['meta']['created'] : // v2 comments
+                            $comment['meta']['postdate'] // v1 comments
```

Comment:
This is specific to only the FS storage and to me a tristate seems much more readable. GCS and S3 got introduced after v2, so they will not encounter v1 comments. And the DB storage still keeps the epoch in a column called "postdate":
https://github.com/PrivateBin/PrivateBin/blob/a93c8ceccb5230eaef569fd9d1f4cd5f8887072a/lib/Data/Database.php#L365

Aside: The order of comments is always controlled by the abstract getOpenSlot method, since comments with the same postdate and parent are allowed.